src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationService.java

```diff
@@ -41,7 +41,8 @@ public void processPendingNotifications() {
 
     @Transactional(readOnly = true)
     public List<Notification> findByFileNameAndService(String fileName, String service) {
-        log.info("Getting notifications for file {}, service {}", fileName, service);
+        final String fileNameForLogPattern = fileName.replaceAll("[\n|\r|\t]", "_");
```

Comment:
```suggestion
        final String fileNameForLogPattern = fileName.replaceAll("[\n|\r|\t]", "");
```

Reply:
You could remove those but please don't change file name. This is not SAS token

Reply:
ok

Reply:
yikes, didn't look at the line :see_no_evil: :speak_no_evil: